DotnetViewComponents/ViewComponents/InsetTextViewComponent.cs

```diff
@@ -0,0 +1,18 @@
+namespace DotnetViewComponents.ViewComponents
+{
+    using System.Collections.Generic;
+    using Microsoft.AspNetCore.Mvc;
+    using DotnetViewComponents.ViewModels;
+
+    public class InsetTextViewComponent : ViewComponent
+    {
+        public IViewComponentResult Invoke(
+            string text,
```

Comment:
would it be better to name this as `contentHtml` if we are rendering it as HTML below - `@Html.Raw(Model.Text)` ?